libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -417,6 +417,37 @@ def ListDiskByLabels(self,
     return self._ListByLabel(
         labels_filter, disk_service_object, filter_union)
 
+  def ListReservedExternalIps(self, zone: str) -> List[str]:
+    """Lists all static external IP addresses that are reserved (i.e. not in
```

Comment:
Mega-nit: I _think_ the first sentence of the docstring is supposed to hold on one line. To give additional details, do it below (in your `the method first converts .... ` block)